pynicotine/downloads.py

```diff
@@ -741,25 +734,21 @@ def get_current_download_file_path(self, transfer):
 
         return self.get_incomplete_download_file_path(transfer.username, transfer.virtual_path)
 
-    def enqueue_folder(self, username, folder_path, download_folder_path=None):
+    def request_folder(self, username, folder_path):
 
         requested_folder = self._requested_folders.get(username, {}).get(folder_path)
 
         if requested_folder is None:
             self._requested_folders[username][folder_path] = requested_folder = RequestedFolder(
-                username, folder_path, download_folder_path
+                username, folder_path
             )
 
-        # First timeout is shorter to get a response sooner in case the first request
-        # failed. Second timeout is longer in case the response is delayed.
-        timeout = 60 if requested_folder.has_retried else 15
-
         if requested_folder.request_timer_id is not None:
             events.cancel_scheduled(requested_folder.request_timer_id)
             requested_folder.request_timer_id = None
 
         requested_folder.request_timer_id = events.schedule(
-            delay=timeout, callback=self._requested_folder_timeout, callback_args=(requested_folder,)
+            delay=15, callback=self._requested_folder_timeout, callback_args=(requested_folder,)
```

Comment:
Often a failure is due to the user being Offline, so this delay is far too long to wait for the error to appear.

Suggest abandoning the `FolderContentsRequest()` in this case (since it is only creating unnecessary network traffic anyway) and calling back immediately with a new method like `requested_folder_offline()` with no delay, if possible.